### PR TITLE
Update ExPlat with more examples

### LINKED ISSUE #33022: Update ExPlat examples
It seems some teams are using a completed experiment as a reference guide. While an ExPlat tips P2 post was written a while back, it may require more example codes (particularly on client side of things).

We should update ExPlat [package](https://github.com/woocommerce/woocommerce/tree/trunk/packages/js/explat) README with more examples as well.

We can also include a simple example in the comment section of [class-experimental-abtest.php](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/react-admin/class-experimental-abtest.php)

### PR DESCRIPTION
**Pull request by moon0326**
### All Submissions:

-   [X] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [X] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #33022  .

This PR updates ExPlat with more examples.

### How to test the changes in this Pull Request:

There's no visual change for the customers.

Please take a look at the changes and make sure they make sense.

### Other information:

-   [X] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [X] Have you successfully run tests with your changes locally?
-   [X] Have you created a changelog file for each project being changed, ie `pnpm nx changelog <project>`?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: packages/js/explat/README.md [hunk 1]
```diff
@@ -40,3 +40,25 @@ import { ExperimentWithAuth } from '@woocommerce/explat';
...
+## Usage with useExperiment
+
+```js
+import { useExperiment } from '@woocommerce/explat';
+
+const DefaultExperience = <div>Hello World</div>;
+
+const TreatmentExperience = <div>Hello WooCommerce!</div>;
+
+const LoadingExperience = <div>⏰</div>;
```
Comment (chihsuan) [inline packages/js/explat/README.md]: Little bit pick. Looks like `LoadingExperience` is not used here.

**Review by chihsuan — DISMISSED** [review]
Thanks for updating the ExPlat docs, Pre-approving! ✅

**Comment by moon0326** [comment]
@chihsuan 

Could you approve it one more time, please 🙇 ? My last commit dismissed your apporval.

**Review by chihsuan — APPROVED** [review]
LGTM!

**Pull request merged**